cmd/juju/ssh/ssh_machine.go

```diff
@@ -535,6 +536,17 @@ func (c *sshMachine) reachableAddressGetter(entity string) (string, error) {
 			return "", errors.Trace(err)
 		}
 	}
+	var sshPort = sshPort
```

Comment:
can we rename the `sshPort` const to `defaultSSHPort`

Reply:
yup I think that would be clearer, thanks Ian!